### ISSUE TITLE
Make followers/followees pages searchable

### ISSUE THREAD

**Issue by mirabilos** [feditext/feditext#102]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I want to write to someone but don’t know their user ID but do know I follow them or they follow me. I go to my profile and then my followers/followees page but then have to scroll/read/guess where.

**Describe the solution you'd like**
I yearn for the ability to Ctrl-F in there. Or whatever one does on iPhone.

**Describe alternatives you've considered**
I suppose I could use the Notes feature of iPhoneOS to keep a list of my contacts, or write something that exposes them as webpage and bookmark that, but…

**Comment by VyrCossont** [comment]
Yeah, this would be good, and it shouldn't be too hard to run a `LIKE %query%` against the on-device database. Likewise, it might be useful to implement fast timeline search the same way.

 I normally use server-side search for this, but most people don't have VyrCossont/mastodon#8, I'm not sure if Mastodon 4.2.0's attempt at account search has scopes like "following" or "followed", and other instance server implementations probably don't have them either.
